netkan/tests/mirrorer.py

```diff
@@ -0,0 +1,60 @@
+import unittest
+
+from netkan.mirrorer import Mirrorer, CkanMirror
+
+
+# I don't know how to test Mirrorer, it needs a queue and archive.org access
+
```

Comment:
This is indeed a challenge. I did build an entire replica of the API for testing NetKAN-bot, which is great but I don't think the value it adds matches the effort required.

Testing wise, we want to think about what is it that is important to test, so that we can change code and have it yell loudly if something changes unexpectedly. Both the important cases are covered, Redistributable + Restricted. But we might want one for handling mods with epochs to ensure our item/title is being produced by CkanMirror correctly.